src/main/scala/uk/gov/hmrc/SbtBobbyPlugin.scala

```diff
@@ -44,57 +48,56 @@ object SbtBobbyPlugin extends AutoPlugin {
+          bobbyConsoleColours.value,
+          deprecatedDependenciesUrl.value,
+          Some(bobbyConfigFile),
+          jsonOutputFileOverride.value
+        )
+      }
+    )
+  )
 
   override lazy val projectSettings = Seq(
     deprecatedDependenciesUrl := None,
     jsonOutputFileOverride := None,
     parallelExecution in GlobalScope := true,
     bobbyViewType := sys.env.get(envKeyBobbyViewType).map(ViewType.apply).getOrElse(Compact),
     bobbyStrictMode := sys.env.get(envKeyBobbyStrictMode).map(_.toBoolean).getOrElse(false),
-    bobbyConsoleColours := sys.env.get(envKeyBobbyConsoleColours).map(_.toBoolean).getOrElse(true),
-    validate := {
-      // Determine nodes to exclude which are this project or dependent projects from this build
-      // Required so multi-project builds with modules that depend on each other don't cause a violation of a SNAPSHOT dependency
-      val extracted = Project.extract(state.value)
-      val internalModuleNodes = buildStructure.value.allProjectRefs.map( p =>
-        extracted.get(projectID in p)
-      ).distinct.map(_.toDependencyGraph)
-
-      // Construct a complete module graph of the project (not plugin) dependencies, piggy-backing off `sbt-dependency-graph`
-      val projectDependencyGraph: ModuleGraph = GraphOps.cleanGraph((moduleGraph in Compile).value, excludeNodes = internalModuleNodes)
-
-      // Retrieve the plugin dependencies. It would be nice to generate these in the same way via the full ModuleGraph, however the
-      // sbt UpdateReport in the pluginData is not rich enough. Seems we have the nodes but not the edges.
-      val pluginDependencies = PluginDependencyResolver.plugins(buildStructure.value).distinct
-
-      // Retrieve just the resolved module IDs, in topologically sorted order
-      val projectDependencies = GraphOps.topoSort(GraphOps.transpose(projectDependencyGraph))
-
-      // Construct a dependency map from each ModuleId -> Sequential list of transitive ModuleIds that brought it in, the tail being the origin
-      val dependencyMap = GraphOps.reverseDependencyMap(projectDependencyGraph, projectDependencies)
-
-      // Retrieve config settings
-      val bobbyConfigFile: ConfigFile = new ConfigFile(System.getProperty("user.home") + "/.sbt/bobby.conf")
-
-      Bobby.validateDependencies(
-        bobbyStrictMode.value,
-        GraphOps.toSbtDependencyMap(dependencyMap), //Use vanilla sbt ModuleIDs
-        projectDependencies.map(_.toSbt),           //Use vanilla sbt ModuleIDs
-        pluginDependencies,
-        scalaVersion.value,
-        bobbyViewType.value,
-        bobbyConsoleColours.value,
-        deprecatedDependenciesUrl.value,
-        Some(bobbyConfigFile),
-        jsonOutputFileOverride.value
-      )
-    }
-  )
+    bobbyConsoleColours := sys.env.get(envKeyBobbyConsoleColours).map(_.toBoolean).getOrElse(true)
+    //Add a useful alias to run bobby validate for compile, test and plugins together (similar to old releases of bobby)
```

Comment:
comment should go between `++ validateSettings` and `++ addCommandAlias` rather than inside `Seq()`?